libs/ledger-live-common/src/hw/connectApp.ts

```diff
@@ -263,7 +271,8 @@ const cmd = ({
           dependencies,
           requireLatestFirmware,
         }: any) =>
-          defer(() => from(getAppAndVersion(transport))).pipe(
+          defer(() => from(getVersion(transport))).pipe(
+            exhaustMap(() => from(getAppAndVersion(transport))),
```

Comment:
Like we discussed, I've never used this operator and it seems unnecessary since `from(getVersion(transport))` will at most return one value. But I don't think it does any harm.